### ISSUE TITLE
Identify the first version of semconv that is interpretable by Weaver

### ISSUE THREAD

**Issue by lquerel** [open-telemetry/weaver#181]
Add this information to the documentation.

**Comment by MadVikingGod** [comment]
TLDR: v1.19.0-v1.26.0 work.

I explored this a bit and found:
from [open-telemetry/semantic-conventions](https://github.com/open-telemetry/semantic-conventions/tree/main/model) these worked:

- [v1.21.0](https://github.com/open-telemetry/semantic-conventions/tree/v1.21.0/model)
- [v1.22.0](https://github.com/open-telemetry/semantic-conventions/tree/v1.22.0/model)
- [v1.23.0](https://github.com/open-telemetry/semantic-conventions/tree/v1.23.0/model)
- [v1.24.0](https://github.com/open-telemetry/semantic-conventions/tree/v1.24.0/model)
- [v1.25.0](https://github.com/open-telemetry/semantic-conventions/tree/v1.25.0/model)
- [v1.26.0](https://github.com/open-telemetry/semantic-conventions/tree/v1.26.0/model)

from [open-telemetry/opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.20.0/semantic_conventions) these work:

- [v1.19.0](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.19.0/semantic_conventions)
- [v1.20.0](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.20.0/semantic_conventions)

v1.18.0 failed with the following error:
```
/weaver registry resolve -r ../opentelemetry-specification/semantic_conventions -f yaml
⠇ Resolving registry `../opentelemetry-specification/semantic_conventions`Diagnostic report:

  × The semantic convention spec is invalid (path_or_url: "../opentelemetry-specification/semantic_conventions/
  │ trace/http.yaml"). groups[2].attributes: data did not match any variant of untagged enum AttributeSpec at line
  │ 135 column 7


Total execution time: 0.100291173s
```
I don't know if this is a simple error to bypass.

**Comment by lquerel** [comment]
I believe the issue comes from `requirement_level: optional` used for one of the attributes. This is not a variant supported by Weaver. My experience with SemConv isn't extensive enough to know if this variant was removed or renamed, but that's probably what happened. If necessary, we could add this variant and issue a warning. @MadVikingGod and @jsuereth What do you think?

**Comment by MadVikingGod** [comment]
I would do an exploration of this. If this is the only problem stopping us from parsing v1.18 and any other before, then yes, I would try to fix it.  But if this just uncovers more issues, it's probably not a path worth pursuing.

**Comment by MadVikingGod** [comment]
With #199 Merged weaver supports v1.13.0 to v1.26.0.  I think we can close this issue.
